### PR TITLE
XRDDEV-2225 API for uploading and downloading configuration parts

### PR DESCRIPTION
**Pull request by justasnortal**
https://nordic-institute.atlassian.net/browse/XRDDEV-2225

### CODE REVIEW

File: src/central-server/admin-service/core/src/main/java/org/niis/xroad/cs/admin/core/service/ConfigurationServiceImpl.java [hunk 1]
```diff
@@ -251,6 +331,49 @@ public Set<DistributedFile> getAllConfigurationFiles(int version) {
                 .collect(toSet());
     }
 
+
+    @Override
+    public void uploadConfigurationPart(ConfigurationSourceType sourceType,
```
Comment (ricardas-buc) [inline src/central-server/admin-service/core/src/main/java/org/niis/xroad/cs/admin/core/service/ConfigurationServiceImpl.java:336]: Maybe consider splitting into upload/download services? both service and unit test are reaching 500 lines

**Review by ricardas-buc — APPROVED** [review]

File: src/central-server/admin-service/core/src/main/java/org/niis/xroad/cs/admin/core/service/ConfigurationServiceImpl.java (hunk 1 above)
Comment (justasnortal) [reply src/central-server/admin-service/core/src/main/java/org/niis/xroad/cs/admin/core/service/ConfigurationServiceImpl.java:213]: Extracted Anchor related code to separate method.

**Comment by justasnortal** [comment]
[rerun ci]

**Comment by justasnortal** [comment]
[rerun ci]

**Pull request merged**